_posts/dev/2015-04-05-api.md

```diff
@@ -1035,6 +1080,34 @@ Missing parameter error
 }
 ```
 
+## getMeetingSummary
+
+Get a summary of events that occurred during a meeting.
+
+**Resource URL:**
+
+- http&#58;//yourserver.com/bigbluebutton/api/v1/getMeetingSummary?[parameters]&checksum=[checksum]
+
+**Parameters:**
+
+{% include api_table.html endpoint="getMeetingSummary" %}
+
+**Example Requests:**
+
+- http&#58;//yourserver.com/bigbluebutton/api/v1/getMeetingSummary?recordID=record123&checksum=1234
+
+**Example Response:**
+
+```xml
+<response>
+  <returncode>SUCCESS</returncode>
+  <meetingSummary>
+  contents of events.xml
```

Comment:
Shouldn't this be the JSON output (in CDATA) from the [bbb_events](https://github.com/bigbluebutton/bbb-events) Gem?

Reply:
It could be done like that. I simple took the json that is returned from the bbb_events Gem and converted it into xml so that there isn't a mix of both.

Reply:
Is the conversion automatic (via a built-in function so there is no maintenance needed as the JSON content expands) and is a one-to-one mapping?

Reply:
Yes, the conversion happens automatically. The events data is stored as raw json in the DB and when it is retrieved it gets converted into xml.